esmvaltool/cmorizers/data/formatters/datasets/osi_450_sh.py

```diff
@@ -21,9 +21,8 @@
     If you also want to cmorize the sh, download everything and create a link
     for OSI-450-nh pointing to the data folder. Both cmorizers will ignore
     files belonging to the other hemisphere.
-
 """
-from .osi_common import OSICmorizer
+from esmvaltool.cmorizers.data.formatters.osi_common import OSICmorizer
 
 
 def cmorization(in_dir, out_dir, cfg, _):
```

Comment:
This script is currently not working because of the call to the cmorization function.

Reply:
Fixed